precode.sh

```diff
@@ -6,38 +6,42 @@
 #   dir2
 #   dir3
 #       dir4
-
+mkdir -p task task/dir1 task/dir2 task/dir3/dir4
 # изменяем текущую директорию на task
-
+cd task
 # создаём пустой файл task/dir2/empty
-
+touch ./dir2/empty
```

Comment:
`./` везде можно опустить, она не нужно
```suggestion
touch dir2/empty
```